specification/quantum/data-plane/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,552 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Quantum Client",
+    "description": "Azure Quantum REST API client",
+    "version": "2019-11-04-preview",
+    "x-ms-code-generation-settings": {
+      "name": "QuantumClient"
+    }
+  },
+  "host": "app-jobscheduler-prod.azurewebsites.net",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Quantum/workspaces/{workspaceName}/jobs": {
```

Comment:
This is a really long path. Is there a strong reason to mirror (what is effectively) the ARM resource instance's path?

Reply:
That's a good question. I think there was, but maybe we just wanted to keep URLs consistent.